Digipost/APIClient.swift

```diff
@@ -433,9 +381,15 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
                         self.deleteRefreshTokensAndLogoutUser()
                     }
             })
-        }else if (oAuthToken == nil) {
+        } else if (oAuthToken == nil) {
             assert(false," something wrong with oauth token")
         } else {
```

Comment:
I have a bit of a hard time figuring out the logic that brings us here… if we have an OAuthToken without a refresh token, it means that we're trying to jump to a higher level oauth token? Why's that? Could we make the code more explicit?